src/translations/en.json

```diff
@@ -2048,11 +2048,29 @@
                   "conversation_engine": "Conversation agent",
                   "conversation_language": "[%key:ui::panel::config::voice_assistants::assistants::pipeline::detail::form::language%]",
                   "language": "Language",
-                  "stt_engine": "Speech to text",
+                  "stt_engine": "Speech-to-text",
                   "stt_language": "[%key:ui::panel::config::voice_assistants::assistants::pipeline::detail::form::language%]",
-                  "tts_engine": "Text to speech",
+                  "tts_engine": "Text-to-speech",
                   "tts_language": "[%key:ui::panel::config::voice_assistants::assistants::pipeline::detail::form::language%]",
                   "tts_voice": "Voice"
+                },
+                "steps": {
+                  "config": {
+                    "title": "Configuration",
+                    "description": "Main configuration of your assistant."
+                  },
+                  "conversation": {
+                    "title": "Conversation agent",
+                    "description": "The conversation agent is the brains of your assistant and will process the incoming text commands."
```

Comment:
```suggestion
                    "description": "The conversation agent is the brains of your assistant. It will process the incoming text commands."
```